code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -217,6 +217,10 @@ var/veil_thickness = CULT_PROLOGUE
 	initialize_cultwords()
 	AppendObjective(/datum/objective/bloodcult_reunion)
 
+
+/datum/faction/bloodcult/minorVictoryText()
+	return "The cult did not manage to summon Nar-Sie, but could achieve most of its objectives and sacrifice the designated victim."
```

Comment:
Better phrasing would be "The cult completed its sacrificial ritual, but not in time to summon Nar-Sie."